### ISSUE TITLE
Why is the return value of this function erfa.rz() unreasonable?

### ISSUE THREAD

**Issue by ying365** [liberfa/pyerfa#68]
void eraRz(double psi, double r[3][3]); When I use this function, the result i get is a matrix contains true and false.

**Comment by mhvk** [comment]
Sorry for the slow reply: when I try in python, it does seem to work fine:
```
In [26]: erfa.rz(np.deg2rad(60.), np.eye(3))
Out[26]: 
array([[ 0.5      ,  0.8660254,  0.       ],
       [-0.8660254,  0.5      ,  0.       ],
       [ 0.       ,  0.       ,  1.       ]])
```

**Comment by ying365** [comment]
>>> import numpy as np
>>> import erfa
>>> erfa.rz(np.deg2rad(60.),np.eye(3))
array([[False, False, False],
       [False, False, False],
       [ True,  True,  True]])

Thank you for your reply. Sorry,  as you can  see , i still can not get the same result. And i  do not know what went wrong.

**Comment by mhvk** [comment]
That is super weird!  Could you let me know which versions of erfa and numpy (and maybe python?) you are using?

**Comment by ying365** [comment]
Erfa is 1.7.2, numpy is 1.19.2, and python is 3.6.5.

**Comment by mhvk** [comment]
Hmm, only the python version is slightly suspicious, but numpy supports it and I don't see what would be in pyerfa that wouldn't. Grasping at straws here, but I assume the two inputs `np.deg2rad(60)` and `np.eye(3)` actually look OK?

**Comment by ying365** [comment]
Maybe i need to rewrite this function in python. My friends still have the same result with me and our python version is different. The inputs that you assume are ok, just an angle and a matrix.

**Comment by mhvk** [comment]
Totally bizarre, still. I wish I could reproduce it, since now it is so difficult to debug. I presume `erfa.rx` and `erfa.ry` give similarly strange output?

On other options: this is of course easy enough to write yourself, but if you use astropy, you can use
```
from astropy.coordinates.matrix_utilities import rotation_matrix
from astropy import units as u

rotation_matrix(60*u.deg, 'z')

# array([[ 0.5      ,  0.8660254,  0.       ],
#        [-0.8660254,  0.5      ,  0.       ],
#        [ 0.       ,  0.       ,  1.       ]])
```

**Comment by ying365** [comment]
Ok, i get it. Thank you for your help. And they have the same strange result with erfa.rz, just including false and true. How incomprehensible !

**Comment by mhvk** [comment]
@ying365 - I have been adding new tests, and it turns out `rx`, `ry`, and `rz` fail on MacOS, while on linux and windows they pass.  Are you running on Mac?

**Comment by mhvk** [comment]
OK, now found what the problem is; fix in #72. Thanks so much for reporting!
